### DISCUSSION TITLE
husky - pre-commit hook exited with code 1 (error)

### DISCUSSION THREAD

**Discussion by SvenMeyer** [PaulRBerg/hardhat-template#74] [Q&A] [answered]
I am using this template for all my new projects and it's a great help - @paulrberg thank you !

However, I just created a new project from the template wanted to commit my changes and get an error message from husky with no comments on the error ...

```
$ git commit -m "feat: initial commit"
➤ YN0000: ┌ Resolution step
➤ YN0000: └ Completed in 1s 761ms
➤ YN0000: ┌ Fetch step
➤ YN0000: └ Completed
➤ YN0000: ┌ Link step
➤ YN0000: └ Completed in 1s 239ms
➤ YN0000: Done in 3s 102ms

husky - pre-commit hook exited with code 1 (error)

```

**Comment by PaulRBerg** [comment, marked as answer]
Hey @SvenMeyer thanks for reporting. This is not an issue with my template - it has to do with Husky's backend. See https://github.com/typicode/husky/issues/1066, https://github.com/typicode/husky/issues/1014 and https://github.com/typicode/husky/issues/651.

The temporary solution for now is to append the git `--no-verify` flag, or its shorthand `-n`.

**Reply by PaulRBerg** [reply]
**Update**: it turns out this was an issue with `lint-staged`: https://github.com/okonet/lint-staged/issues/1032

[Upgrading](https://github.com/paulrberg/solidity-template/commit/b3f8674e08202fa7904f67cd34e251c7caa30628) to `lint-staged` v11.2.5 fixed the issue. @SvenMeyer you can go back to committing without the `--no-verify` flag.
